lib/modules/uscore_v3.1.0/access_verify_unrestricted_sequence.rb

```diff
@@ -7,36 +7,35 @@ class ONCAccessVerifyUnrestrictedSequence < SequenceBase
 
       description 'Verify that all resource types can be accessed by apps with appropriate scopes.'
       test_id_prefix 'AVU'
-
       details %(
-
-        The following are required to be seen:
+        This test ensures that apps have full access to USCDI resources if granted access by the tester.
+        The tester must grant access to the following resources during the SMART Launch process,
+        and this test ensures they all can be accessed:
 
           * AllergyIntolerance
-
           * CarePlan
-
           * CareTeam
-
           * Condition
-
           * Device
-
           * DiagnosticReport
-
           * DocumentReference
-
           * Goal
-
           * Immunization
-
           * MedicationRequest
-
           * Observation
-
           * Procedure
 
-
+        For each of the resources that can be mapped to USCDI data class or elements, this set of tests
```

Comment:
```suggestion
        For each of the resource types that can be mapped to USCDI data class or elements, this set of tests
```

changing to resource types for consistency